### DISCUSSION TITLE
European Association for Cancer Research (EACR) Highlights the power of IBEX Imaging

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#178] [Announcements]
In its Highlights in Cancer Research, [June 2024 edition](https://magazine.eacr.org/highlights-in-cancer-research-june-2024/5/), the EACR calls attention to work illustrating the power of IBEX Imaging: "Multi-omic profiling of follicular lymphoma reveals changes in tissue architecture and enhanced stromal remodeling in high-risk patients".
 
This work illustrates how IBEX Imaging is just as useful when your focus is on applied science as when it is on basic science.
 
Question: Should I pursue an Applied or Basic research career? [The “ultimate” choice between A or B]
Answer: 42 [it worked in "The Hitchhiker's Guide to the Galaxy" and the [Tiff header](https://en.wikipedia.org/wiki/TIFF#Byte_order), so why not here]

**Comment by radtkea** [comment]
Thanks so much for posting Ziv! This work was the WHY that created the HOW of IBEX. So grateful for all your support, encouragement, creativity, and friendship over the years! Could not have done it without you!

In answer to your question:
**C** Live your life as an offering! But also **A**pplied/translational.

Love your musings:)
